src/Model/Sortable/Sortable.php

```diff
@@ -2,49 +2,12 @@
 
 namespace Knp\DoctrineBehaviors\Model\Sortable;
 
+/**
+ * Trait Sortable
```

Comment:
Well, we could to add more description here. Btw, it also needs some descriptions for docs. There is missing behaviour description.